spec/fortnox/api/validators/invoice_spec.rb

```diff
@@ -34,9 +28,9 @@
     include_examples 'validates length of string', :your_reference, 50
     include_examples 'validates length of string', :zip_code, 1024
 
-    include_examples 'validates inclusion of', :administration_fee, 0, 99_999_999_999.0
-    include_examples 'validates inclusion of', :currency_rate, 0, 999_999_999_999_999.0
-    include_examples 'validates inclusion of', :currency_unit, 0, 999_999_999_999_999.0
-    include_examples 'validates inclusion of', :freight, 0, 99_999_999_999.0
+    include_examples 'validates inclusion of number', :administration_fee, 0, 99_999_999_999.0
+    include_examples 'validates inclusion of number', :currency_rate, 0, 999_999_999_999_999.0
+    include_examples 'validates inclusion of number', :currency_unit, 0, 999_999_999_999_999.0
+    include_examples 'validates inclusion of number', :freight, 0, 99_999_999_999.0
```

Comment:
Really need to make a `validates coverage of number` version of this ... I'll put it on my todo.